build_all_images.sh

```diff
@@ -68,6 +68,14 @@ function build_bundle_image {
 	local test_hotfix_url=$(get_string $(yq "${query}".test_hotfix_url < bundles.yml))
 	local test_installed_patch=$(get_string $( yq "${query}".test_installed_patch < bundles.yml))
 
+	if is_release_candidate && is_dxp_release
+	then
+		bundle_url="releases-cdn.liferay.com/dxp/release-candidates/${version}/"$(curl -fsSL "https://releases-cdn.liferay.com/dxp/release-candidates/${version}/.lfrrelease-tomcat-bundle")
```

Comment:
@kiwm is possible quote everything just once like `bundle_url="releases-cdn.liferay.com/dxp/release-candidates/${version}/$(curl -fsSL "https://releases-cdn.liferay.com/dxp/release-candidates/${version}/.lfrrelease-tomcat-bundle")"` ?